src/config/2020.json

```diff
@@ -220,6 +220,16 @@
       "avatar_url": "https:\/\/avatars0.githubusercontent.com\/u\/1120896?v=4&s=200",
       "github": "rviscomi",
       "twitter": "rick_viscomi"
+    },
+    "dsadhanala": {
+      "name": "Durga Prasad Sadhanala",
+      "teams": [
+        "developers"
+      ],
+      "avatar_url": "https:\/\/avatars3.githubusercontent.com\/u\/5795823?s=200&v=4",
+      "website": "https:\/\/web-ui.dev",
+      "github": "dsadhanala",
+      "twitter": "dsadhanala"
```

Comment:
Any chance you could move this up above Ricks? I try to keep it in first name alphabetical order to make it easier to find, avoid any thoughts of ordering and satisfy my OCD (mostly the last one to be honest!)

Reply:
no problem, I'll move this. thanks for the review!